ext/storage_sources/local_store/local_store.c

```diff
@@ -304,6 +307,34 @@ local_flush_free(LOCAL_FLUSH_ITEM *flush)
     }
 }
 
+/*
+ * local_get_directory --
+ *     Return a copy of a directory name after verifying that it is a directory.
+ */
+static int
+local_get_directory(const char *s, ssize_t len, char **copy)
+{
+    struct stat sb;
+    int ret;
+    char *dirname;
+
+    if (len == -1)
+        len = (ssize_t)strlen(s);
+    dirname = strndup(s, (size_t)len + 1); /* Room for null */
+    if (dirname == NULL)
+        return (ENOMEM);
+    ret = stat(dirname, &sb);
```

Comment:
Do we need to make sure that any python test or other Csuite test using this is not Windows?

Reply:
The ext/storage_source/local_store/ directory is configured only to be built on POSIX.  The only test that uses local_store right now, "skips" it if local_store is not built, we'll need to do that for any new tests that use local_store - it's easy to do.  I suspect if we have any Csuite tests in the future (like abort_flush_tier), we'll enable that for only POSIX.  So I think we're covered now.